internal/executor/taskfilter/taskfilter.go

```diff
@@ -63,8 +63,10 @@ func extractLabels(s string) (result []string) {
 	return
 }
 
-func matchLabels(soughtLabels, actualLabels []string) (numMatchedLabels int) {
-	for _, soughtLabel := range soughtLabels {
+func containsAll(desiredLabels, actualLabels []string) bool {
```

Comment:
How about still switching places of the arguments so the slice where the search will be performed will be first.

Reply:
See ac0909fc56b31e7ca3016e3aa9f0600eb120521e.